donate/thunderbird/templates/fragments/donate_form_disclaimer.html

```diff
@@ -9,11 +9,12 @@
 {% endblock %}
 
 {% block faq_contact %}
-    {% blocktrans with faq_url='/faq' email='give@thunderbird.net' trimmed %}
+    {% blocktrans with faq_url='/faq' email='thunderbird-donate@mozilla.org' trimmed %}
```

Comment:
This is email change is temporary... wondering if updating the `email` var here is okay or this will affect localization?